### PR TITLE
[zh-cn]: Update translation of Event

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

API and Docs list was outdated.

### Additional details

None.

### Related issues and pull requests

None.

### CODE REVIEW

**Comment by yin1999** [comment]
Hi @JasonLamv-t, sorry for modified the description of this document without notification. Since this document is quite important, I'd like to make it a full update ;)

**Comment by JasonLamv-t** [comment]
> Hi @JasonLamv-t, sorry for modified the description of this document without notification. Since this document is quite important, I'd like to make it a full update ;)

It doesn't matter. I just noticed that many links in the list part are invalid, and I did not carefully proofread and update the description part. Thank you for your addition. O.o

File: files/zh-cn/web/api/event/index.md [hunk 1]
```diff
@@ -82,72 +75,58 @@ slug: Web/API/Event
...
+  - : A boolean value indicating whether the given event will bubble across through the shadow root into the standard DOM. Use {{domxref("Event.composed", "composed")}} instead.
+  - : 一个布尔值，指示给定事件是否会穿过影子根节点冒泡到标准 DOM。请改用 {{domxref("Event.composed", "composed")}}。
 
-## 方法
+## 实例方法
 
-- {{domxref("Event.createEvent()")}} {{deprecated_inline}}
-  - : 创建一个新事件，如果使用此方法创建事件，则必须调用其自身的 `initEvent()` 方法，对其进行初始化。
 - {{domxref("Event.composedPath()")}}
   - : 返回事件的路径（将在该对象上调用监听器）。如果阴影根节点 (shadow root) 创建时 {{domxref("ShadowRoot.mode")}} 值为 closed，那么路径不会包括该根节点下阴影树 (shadow tree) 的节点。
```
Comment (yin1999) [inline files/zh-cn/web/api/event/index.md:118]: see: https://github.com/mdn/translated-content/pull/13432#discussion_r1208745293

Comment (yin1999) [reply files/zh-cn/web/api/event/index.md:115]: resolved in #13432, the translation is suitable here ;)

File: files/zh-cn/web/api/event/index.md [hunk 2]
```diff
@@ -160,7 +138,3 @@ slug: Web/API/Event
 - 可用的事件类型：[Event 参考](/zh-CN/docs/Web/Reference/Events)
```
Comment (yin1999) [inline files/zh-cn/web/api/event/index.md:138]:
```suggestion
- 可用的事件类型：[Event 参考](/zh-CN/docs/Web/Events)
```

File: files/zh-cn/web/api/event/index.md [hunk 3]
```diff
@@ -160,7 +138,3 @@ slug: Web/API/Event
...
 - [Event 目标的比较](/zh-CN/docs/Web/API/Event/Comparison_of_Event_Targets)（目标 `target` vs 当前目标 `currentTarget` vs 相关目标 `relatedTarget` vs 初始目标 `originalTarget`）
 - [创建和触发自定义事件](/zh-CN/docs/Web/Guide/DOM/Events/Creating_and_triggering_events)
```
Comment (yin1999) [inline files/zh-cn/web/api/event/index.md:140]:
```suggestion
- [创建和触发自定义事件](/zh-CN/docs/Web/Events/Creating_and_triggering_events)
```

**Review by yin1999 — COMMENTED** [review]
All the suggestions are listed, please take a look on those.

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**